app/assets/stylesheets/components/_form-elements.scss

```diff
@@ -92,6 +92,10 @@
   &:before {
     display: none;
   }
+
+  &:after {
+    display: none;
+  }
 }
```

Comment:
removes a little pointer UI doohickey. didn't seem to be needed for any other usages i could find but it is a global change.